rts/Lua/LuaSyncedRead.cpp

```diff
@@ -8999,6 +9001,19 @@ int LuaSyncedRead::GetRadarErrorParams(lua_State* L)
 	return 3;
 }
 
+/***
```

Comment:
would be nice to move the implementation among other feature-related ones

Reply:
Done!